src/server/scripts/Kalimdor/zone_mulgore.cpp

```diff
@@ -428,7 +428,47 @@ class npc_agitated_earth_spirit_36845 : public CreatureScript
         return new npc_agitated_earth_spirit_36845AI(creature);
     }
 };
+//FIX THE QUEST 27066 MULGORE priest quest 
+enum eQuest27066
+{
+	NPC_WOUNDED_BRAVE = 45199,
+	SPELL_FLASH_HEAL_2061 = 2061,
+	NPC_KILLCREDIT = 44175,
```

Comment:
if i look on wiki: there are 2 quest that work with this npc.. quest 27066 and quest 27067
so the brave npc must accept both spell for KillMonsterKredit
spell 2061 flash heal priest
spell 774 Rejuvenating druid
(i hope i have seen it correct.. many quest have changed metween 403 and 434)